src/display/editor/editor.js

```diff
@@ -926,6 +926,11 @@ class AnnotationEditor {
     if (!tooltip.parentNode) {
       button.append(tooltip);
     }
+
+    const element = this.getImageForAltText();
+    if (element) {
+      element.setAttribute("aria-describedby", tooltip.id);
+    }
```

Comment:
Nit: This could perhaps be a tiny bit shorter?
```suggestion
    element?.setAttribute("aria-describedby", tooltip.id);
```